mmstudio/src/main/java/org/micromanager/internal/MMROIManager.java

```diff
@@ -145,7 +154,15 @@ private Rectangle updateROI(Roi roi) {
     * Set the ROI to the center quadrant of the current ROI.
     */
    public void setCenterQuad() {
-      ImagePlus curImage = WindowManager.getCurrentImage();
+      DataViewer dv = studio_.displays().getActiveDataViewer();
+      DisplayWindow dw = null;
+      if (dv instanceof DisplayWindow) {
+         dw = (DisplayWindow) studio_.displays().getActiveDataViewer();
+      }
+      ImagePlus curImage = null;
+      if (dw != null) {
+         curImage = dw.getImagePlus();
+      }
```

Comment:
Would be nice to factor this out as a private method, since it is an exact duplicate of the other change.

Reply:
Fully agreed.  All done.